examples/tools/scm/git/capture_scm/git_capture_scm.rst

```diff
@@ -77,26 +71,40 @@ please create a folder outside of the ``examples2`` repository, and copy the con
     # Finally create the package
     $ conan create .
     ...
-    hello/0.1: WARN: Current commit 8e8764c40bebabbe3ec57f9a0816a2c8e691f559 doesn't exist in remote origin
-    This revision will not be buildable in other computer
-    hello/0.1: Obtained URL: <local-path>/capture_scm and 8e8764c40bebabbe3ec57f9a0816a2c8e691f559
-    hello/0.1: Copied 1 '.yml' file: conandata.yml
+    ======== Exporting recipe to the cache ========
+    hello/0.1: Exporting package recipe: /examples2/examples/tools/scm/git/capture_scm/conanfile.py
+    hello/0.1: Calling export()
+    hello/0.1: RUN: git status . --short --no-branch --untracked-files
+    hello/0.1: RUN: git rev-list HEAD -n 1 --full-history -- "."
+    hello/0.1: RUN: git remote -v
+    hello/0.1: RUN: git branch -r --contains b0809960b96002053d954cec112f1fa4a4d306d2
     hello/0.1: Copied 1 '.py' file: conanfile.py
+    hello/0.1: Copied 1 '.yml' file: conandata.yml
     ...
-    hello/0.1: Cloning sources from: {'commit': '8e8764c40bebabbe3ec57f9a0816a2c8e691f559', 'url': '<local-path>/capture_scm'}
+    hello/0.1: Cloning git repo
+    hello/0.1: RUN: git clone "<hidden>"  "."
+    hello/0.1: Checkout: b0809960b96002053d954cec112f1fa4a4d306d2
+    hello/0.1: RUN: git checkout b0809960b96002053d954cec112f1fa4a4d306d2
 
 Let's explain step by step what is happening:
 
-- When the recipe is exported to the Conan cache, the ``export()`` method executes, running ``scm_url, scm_commit = git.get_url_and_commit()``. See the :ref:`Git reference<conan_tools_scm_git>` for more information about these methods.
+- When the recipe is exported to the Conan cache, the ``export()`` method executes, ``git.coordinates_to_conandata()``,
+  which stores the Git URL and commit in the ``conandata.yml`` file by internally calling ``git.get_url_and_commit()``.
+  See the :ref:`Git reference<conan_tools_scm_git>` for more information about these methods.
 - This obtains the URL of the repo pointing to the local ``<local-path>/capture_scm`` and the commit ``8e8764c40bebabbe3ec57f9a0816a2c8e691f559``
 - It warns that this information will **not** be enough to re-build from source this recipe once the package is uploaded to the server and is tried to be built from source in other computer, which will not contain the path pointed by ``<local-path>/capture_scm``. This is expected, as the repository that we created doesn't have any remote defined. If our local clone had a remote defined and that remote contained the ``commit`` that we are building, the ``scm_url`` would point to the remote repository instead, making the build from source fully reproducible.
```

Comment:
But this doesn't happen anymore? There are no such warnings about this, what has changed?

Reply:
This is what I locally get 
```
$ conan create .

======== Exporting recipe to the cache ========
hello/0.1: Exporting package recipe: /Users/ruben/coding/myfolder/conanfile.py
hello/0.1: Calling export()
hello/0.1: RUN: git status . --short --no-branch --untracked-files
hello/0.1: RUN: git rev-list HEAD -n 1 --full-history -- "."
hello/0.1: RUN: git remote -v
hello/0.1: RUN: git branch -r --contains cb7815a58529130b49da952362ce8b28117dee53
hello/0.1: RUN: git fetch origin --dry-run --depth=1 cb7815a58529130b49da952362ce8b28117dee53
hello/0.1: WARN: Current commit cb7815a58529130b49da952362ce8b28117dee53 doesn't exist in remote origin
This revision will not be buildable in other computer
hello/0.1: RUN: git rev-parse --show-toplevel
hello/0.1: Copied 1 '.py' file: conanfile.py
hello/0.1: Copied 1 '.yml' file: conandata.yml
hello/0.1: Exported to cache folder: /Users/ruben/.conan2/p/hello237d6f9f65bba/e
hello/0.1: Exported: hello/0.1#b9ef93c37197d05481ee699119090d48 (2024-04-05 16:06:40 UTC)
```
 isn't it what these lines describe?